fileWithPRComments.js

```diff
@@ -0,0 +1,8 @@
+/* global QamineUtils */
+
+"use strict";
+
+(function($, window) {
```

Comment:
![Codacy](http://release.k8s.dev.codacy.org/assets/images/favicon.png) Issue found: [Move the invocation into the parens that contain the function.](http://release.k8s.dev.codacy.org/manual/devk8scodacytestuser/CheckPrCommentsPublicProjectDev/pullRequest?prid=207)